dashboard/legacy/middleware/files_api.rb

```diff
@@ -268,7 +268,6 @@ def get_file(endpoint, encrypted_channel_id, filename, code_projects_domain_root
     metadata = result[:metadata]
     abuse_score = [metadata['abuse_score'].to_i, metadata['abuse-score'].to_i].max
     not_found if abuse_score >= SharedConstants::ABUSE_CONSTANTS.ABUSE_THRESHOLD && !can_view_abusive_assets?(encrypted_channel_id)
-    not_found if profanity_privacy_violation?(filename, result[:body]) && !can_view_profane_or_pii_assets?(encrypted_channel_id)
```

Comment:
does this mean we aren't checking for profanity at all?

Reply:
Oh gotcha! Could also update or remove that function (should it always return false now?)

Reply:
Yes, we should. But I am proposing to do so in a follow-up PR because there are other places it is called that other teams may be impacted. See comment in investigation PR: https://github.com/code-dot-org/code-dot-org/pull/65463/files#r2056956932

Reply:
We switched to just `find_failure` here: https://github.com/code-dot-org/code-dot-org/pull/65630